### PR TITLE
Move to ESM

### PR DESCRIPTION
**Pull request by UltimateLobster**

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "extends": ["noamokman", "prettier"],
+  "extends": ["@osskit", "prettier"],
```
Comment (ItamarKieslerFabric) [inline .eslintrc:2]: can remove the prettier config, it's not needed anymore

File: package.json [hunk 2]
```diff
@@ -15,24 +22,23 @@
...
   },
   "dependencies": {
-    "axios": "^0.21.1"
+    "axios": "^0.27.2"
   },
   "devDependencies": {
-    "@types/node": "^16.3.2",
+    "@osskit/eslint-config": "^1.0.3",
+    "@osskit/prettier-config": "^0.0.1",
+    "@types/node": "^18.0.0",
```
Comment (ItamarKieslerFabric) [inline package.json:30]: Not directly related to ESM, but might consider upgrading node 16 to 18 everywhere

**Review by ItamarKieslerFabric — APPROVED** [review]

File: tsconfig.json [hunk 3]
```diff
@@ -2,8 +2,8 @@
   "compilerOptions": {
     "target": "ESNext",
     "lib": ["esnext"],
-    "module": "CommonJS",
-    "moduleResolution": "Node",
+    "module": "ES2020",
```
Comment (SharonGrossman) [inline tsconfig.json:5]: we can use ESNext

**Review by noamokman — APPROVED** [review]

File: .github/workflows/lint.yml [hunk 4]
```diff
@@ -19,6 +19,6 @@ jobs:
       - name: Use Node.js
         uses: actions/setup-node@v2
         with:
-          node-version: "16.x"
+          node-version: "18.x"
       - run: yarn
-      - run: yarn lint
\ No newline at end of file
+      - run: yarn lint
```
Comment (SharonGrossman) [inline .github/workflows/lint.yml:24]: we can add the lint:base script
and use it here

**Review by SharonGrossman — APPROVED** [review]

**Pull request merged**